opentelekomcloud/acceptance/nat/import_opentelekomcloud_nat_gateway_v2_test.go

```diff
@@ -0,0 +1,27 @@
+package acceptance
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/common"
+)
+
+func TestAccNatV2Gateway_importBasic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckNatV2GatewayDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNatV2GatewayBasic,
+			},
+			{
+				ResourceName:      resourceGatewayName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
```

Comment:
Please, add this test in resource test don't produce more `acceptance_test` files for existing resources

Reply:
good point